ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -1,8 +1,8 @@
 {# Leaf connected_endpoint interfaces #}
 ethernet_interfaces:
 {% for endpoint_key in connected_endpoints | arista.avd.natural_sort %}
-{%     for connected_endpoint in connected_endpoints[endpoint_key] | arista.avd.natural_sort %}
-{%         for adapter in connected_endpoints[endpoint_key][connected_endpoint].adapters %}
+{%     for connected_endpoint in connected_endpoints[endpoint_key] | arista.avd.convert_dicts('endpoint') | arista.avd.natural_sort %}
```

Comment:
These templates are consuming data from the `logic.j2`, so you should only implement the conversion in logic. You don't need to change the output of logic.j2: `connected_endpoints`.

Reply:
Fixed